src/Mod/Fem/femtaskpanels/task_result_mechanical.py

```diff
@@ -503,13 +503,15 @@ def result_selected(self, res_type, res_values, res_unit, res_title):
 
         if len(plt.get_fignums()) > 0:
             plt.close()
+        plt.ioff()
         plt.hist(res_values, bins=50, alpha=0.5, facecolor="blue")
         plt.xlabel(res_unit)
         plt.title(translate("FEM","Histogram of {}").format(res_title))
         plt.ylabel(translate("FEM","Nodes"))
         plt.grid(True)
         fig_manager = plt.get_current_fig_manager()
-        fig_manager.window.setWindowFlags(QtCore.Qt.WindowStaysOnTopHint)    # stay ontop
+        fig_manager.window.setParent(FreeCADGui.getMainWindow())
+        fig_manager.window.setWindowFlag(QtCore.Qt.Tool)    # stay ontop
```

Comment:
Can you please add a comment? For me as non-Pythionist it is not clear why setting a flag "QtCore.Qt.Tool" puts the window on top

Reply:
Nothing to do with Python. ;) It's pure Qt Window Flag : https://doc.qt.io/qt-5/qt.html#WindowType-enum
Will add a comment too.